src/Conflux.ts

```diff
@@ -324,56 +324,43 @@ export default class Conflux {
     r: string;
   }> {
     const paths = splitPath(path);
-    let offset = 0;
     const message = Buffer.from(messageHex, "hex");
-    const toSend: Buffer[] = [];
-    let response;
+    const pathBuffer = Buffer.alloc(1 + paths.length * 4);
```

Comment:
recommend put these logic in a util, this should also been used in other commands